packages/services/carbon.yml

```diff
@@ -0,0 +1,116 @@
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@v1/carbon-resources.schema.json
+library:
+  id: ibmdotcom-services
+  name: IBM.com Services
+  description: IBM.com ES6 service classes
+  noIndex: true
+  externalDocsUrl: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/services
+assets:
+  analytics-api:
+    name: AnalyticsAPI
+    description: Analytics API class with methods for firing analytics events on IBM.com.
+    type: function
+    platform: web
+    status: stable
+    noIndex: true
+    externalDocsUrl: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/services/AnalyticsAPI.html
+    framework: vanilla
+    tags:
+      - service
+  kaltura-player-api:
+    name: KalturaPlayerAPI
+    description: KalturaPlayerAPI class with methods of checking script state and embed media meta data and api data.
+    type: function
+    platform: web
+    status: stable
+    noIndex: true
+    externalDocsUrl: https://carbon-design-system.github.io/carbon-for-ibm-dotcom/services/KalturaPlayerAPI.html
+    framework: vanilla
+    tags:
+      - service
+  ddo-api:
+    name: DDOAPI
+    description: DDO API class with methods of fetching search results for ibm.com.
```

Comment:
Can you do a find-all for ` ibm.com` to update to ` IBM.com` in descriptions? Since these descriptions show up in the catalog list view, we don't want to have lowercase `ibm.com` when not in a URL.

Reply:
Done!